generators/app/templates/src/scss/margins.scss

```diff
@@ -1,15 +1,17 @@
+/* stylelint-disable selector-class-pattern  */
 $breakpoint-tablet: 768px;
 $breakpoint-desktop: 1024px;
-$scale: 10;
 
 @media (max-width: $breakpoint-tablet) {
-  $scale: 5;
+  $bt-scale: 5;
 }
 
 @media (min-width: $breakpoint-desktop) {
-  $scale: 10;
+  $bd-scale: 10;
```

Comment:
since we're here, please change this to 5

Reply:
![tenor-203177749](https://user-images.githubusercontent.com/36163179/56812781-f2474300-6811-11e9-8cb4-3c8abcab9c8e.gif)